src/SDKs/ManagementPartner/Management.ManagementPartner/generate.ps1

```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "managementpartner/resource-manager" -SdkDirectory "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)" -PowershellInvoker  -AutoRestVersion "latest"
```

Comment:
@jeffrey-ACE Please remove the `-SdkDirectory` argument here and generate again.
Apologize for the confusion, will fix the example soon.